druid/src/lens.rs

```diff
@@ -290,3 +343,104 @@ impl<T: Clone, U: Clone, B> Clone for Then<T, U, B> {
         }
     }
 }
+
+#[derive(Debug, Copy, Clone)]
+pub struct Deref;
```

Comment:
if this can appear in documentation I would give it a doc string.

Reply:
Good catch!